packages/samples/vuejs/.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
+  "recommendations": ["johnsoncodehk.volar", "johnsoncodehk.vscode-typescript-vue-plugin"]
```

Comment:
Are those the default recommendations from vue?

Reply:
yeah!